code/src/components/NameQuestion.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export const NameQuestion = ({ name, onNameChange }) => {
```

Comment:
I really liked that you used an object as your prop instead of just writing `(props)`. This makes the component way cleaner and easier to read 👍 
(You did this on all components, so this positive comment is for all those moments 😆 )